projects/storefrontlib/src/cms-components/configurator/commons/config-overview-form/config-overview-form.component.ts

```diff
@@ -28,7 +28,29 @@ export class ConfigOverviewFormComponent implements OnInit {
       .pipe(
         switchMap(owner =>
           this.configuratorCommonsService.getOrCreateConfiguration(owner)
+        ),
+        take(1),
+        switchMap(configuration =>
+          this.configuratorCommonsService.getConfigurationWithOverview(
+            configuration
+          )
         )
       );
   }
+
+  hasAttributes(configuration: Configurator.Configuration): boolean {
+    if (
+      configuration.overview === undefined ||
+      configuration.overview === null
+    ) {
+      return false;
+    }
+
+    for (let g = 0; g < configuration.overview.groups.length; g++) {
```

Comment:
We should do this in functional style:     return (
      configuration.overview.groups.filter(group => group.attributes.length > 0)
        .length > 0